docs/admin/system-information.rst

```diff
@@ -832,11 +832,22 @@ Description of checked node settings
 Recovery expected data nodes
 ............................
 
+.. raw:: html
+
+  <span id="recovery-expected-nodes"></span>
```

Comment:
This is for the existing bitly links to point to the correct section in the document (see PR9397)

Reply:
I know to little about `rst`, is this conform with how we do it elsewhere?
cc: @amotl

Reply:
Hi. If there are existing documents which need an anchor reference to exist like this, and we can do nothing about it, but want to fix it, I think there is no other solution native to reStructuredText.

Reply:
The link in question is https://cr8.is/d-node-check-1

Reply:
Would the proper way not be to fix `https://cr8.is/d-node-check-1` to point to `https://crate.io/docs/crate/reference/en/5.2/admin/system-information.html#recovery-expected-data-nodes` ?